Fix install issue on ubuntu 64-bit hosts

On a 64 bit ubuntu host there is no /lib64 dir which triggered the
fact that Makefile is not correct for install test.

Signed-off-by: Jason Wessel <jason.wessel@windriver.com>

---
 libntfs-3g/Makefile.am |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/libntfs-3g/Makefile.am
+++ b/libntfs-3g/Makefile.am
@@ -56,15 +56,15 @@ endif
 # And create ldscript or symbolic link from /usr
 install-exec-hook: install-rootlibLTLIBRARIES
 if INSTALL_LIBRARY
-	if [ ! "$(rootlibdir)" -ef "$(libdir)" ]; then \
+	if [ ! "$(DESTDIR)$(rootlibdir)" -ef "$(DESTDIR)/$(libdir)" ]; then \
 		$(MV) -f "$(DESTDIR)/$(libdir)"/libntfs-3g.so* "$(DESTDIR)/$(rootlibdir)";  \
 	fi
 if GENERATE_LDSCRIPT
-	if [ ! "$(rootlibdir)" -ef "$(libdir)" ]; then \
+	if [ ! "$(DESTDIR)/$(rootlibdir)" -ef "$(DESTDIR)/$(libdir)" ]; then \
 		$(install_sh_PROGRAM) "libntfs-3g.script.so" "$(DESTDIR)/$(libdir)/libntfs-3g.so"; \
 	fi
 else
-	if [ ! "$(rootlibdir)" -ef "$(libdir)" ]; then \
+	if [ ! "$(DESTDIR)/$(rootlibdir)" -ef "$(DESTDIR)/$(libdir)" ]; then \
 		$(LN_S) "$(rootlibdir)/libntfs-3g.so" "$(DESTDIR)/$(libdir)/libntfs-3g.so"; \
 	fi
 endif
